ocrd_browser/view/base.py

```diff
@@ -249,7 +249,11 @@ class FileGroupModel(Gtk.ListStore):
     def __init__(self, document: Document):
         super().__init__(str, str, str, str)
         for group, mime in document.file_groups_and_mimetypes:
-            self.append(('{}|{}'.format(group, mime), group, mime, MIME_TO_EXT.get(mime, '.???')))
+            if mime == 'text/html':
+                ext = '.html'
+            else:
+                ext = MIME_TO_EXT.get(mime, '.???')
```

Comment:
Or maybe we should add "text/html" to `ocrd_utils.constants.MIME_TO_EXT`?

Reply:
> Or maybe we should add "text/html" to `ocrd_utils.constants.MIME_TO_EXT`?

Thought about that – not sure TBH. (Might have implications elsewhere.) @kba?